client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -208,7 +207,7 @@ endif()
 # NOTE: Platform specific compilation flags should be set 
 # in the corresponding platform CMake script(s).
 if (NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+    list(APPEND KAA_COMPILE_OPTIONS -fPIC)
```

Comment:
Have you tested with non-msvc compilers? `list(APPEND` is not suitable for compiler flags as it inserts `;` and that causes cmake interpret `-fPIC` as a name for executable.